lib/msal-common/src/utils/Constants.ts

```diff
@@ -401,3 +401,38 @@ export enum JsonTypes {
     Jwt = "JWT",
     Jwk = "JWK"
 }
+
+export enum SilentTokenRetrievalStrategy {
+    /*
+     * acquireTokenSilent will first look in the cache before going to the network with the existing refresh token.
+     * The refresh token will be renewed if it is expired.
+     * (existing default behavior)
+     */
+    CacheAndNetwork = 0, // 0 is falsy, is equivalent to not passing in a SilentTokenRetrievalStrategy
+    /*
+     * acquireTokenSilent will only look in the cache, and will not go to the network with the existing refresh token.
+     * The refresh token will not be renewed if it is expired.
+     */
+    CacheOnly = 1,
+    /*
+     * acquireTokenSilent will first look in the cache before going to the network with the existing refresh token.
+     * The refresh token will not be renewed if it is expired.
```

Comment:
AcquireTokenSilent won't be able to renew the RT if is expired. interaction is required.

Reply:
On the web, we can attempt to silently renew the rt using a hidden iframe and prompt=none.